app/views/admin/users/_member_index.html.erb

```diff
@@ -32,9 +32,12 @@
                 <div>
                   <%= render "admin/users/index/status_indicator", user: user %>
                 </div>
+                <!-- %=
+                # TODO: [@forem/admin-experience] Uncomment this chunk of code containing role badges once we finalize which roles to include.
                 <span class="c-indicator ml-2 crayons-hover-tooltip crayons-hover-tooltip--right" data-tooltip="<%= format_role_tooltip(user) %>">
                   <%= cascading_high_level_roles(user) %>
```

Comment:
```suggestion
                  <%#= cascading_high_level_roles(user) %>
```

Without the `<%#=` we will still be calling the `cascading_high_level_roles` method and rendering this information to the response document (but within an HTML comment).